phpstan.neon.dist

```diff
@@ -2,6 +2,8 @@ parameters:
     level: 1
     paths:
         - src
+    bootstrapFiles:
+        - classAliases.php
```

Comment:
thanks! please put the alias file into a `.phpstan` subfolder so we don't pollute the root directory.

and then we can the .phpstan folder to the .gitattributes as export-ignore so installations dont get the file.